Analysis script template.R

```diff
@@ -0,0 +1,261 @@
+
+
+# Variable defined to keep track of simulation time (delete afterwards)
+init <- Sys.time()
+
+# Control variables of simulation
+npats <- 500 # Number of patients in simulation
+tx.cost <- 0 # Total treatment cost
+
+treateff.start <- 1 # Cycle in which treatment effect starts
+treateff.end <- 4 # Cycle in which treatment effect ends
+treateff.decline <- 2 # Cycle in which treatment effect strats to decline linearly
+
+treateff.hba1c <- -1.82 # Treatment effect on HbA1c (in absolute %-points HbA1c)
+treateff.hdl <- 0 # Treatment effect on HDL-choleresterol (absolute effect, which unit??)
+treateff.ldl <- 0 # Treatment effect on LDL-choleresterol (absolute effect, which unit??)
+
+sim.vars <- list(npats, tx.cost, mget(apropos('treateff.')))
```

Comment:
Please explain line 18.

Reply:
The sim.vars object is made just to collect all the parameters that define the simulation into one object, which is then also saved with the output of the simulation. This way, if you have multiple output files you always have the information on the relevant input parameters that were used to produce the output.

As I expect more treatment effect parameters to be included/ to circumvent having to define all treatment effect parameters, I just include all objects whose name includes 'treatefff.'.